starter-code/InvestingPortfolio/app/src/main/java/drewmahrt/generalassemb/ly/investingportfolio/MainActivity.java

```diff
@@ -57,13 +61,46 @@ protected void onCreate(Bundle savedInstanceState) {
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
-        mPortfolioRecyclerView = (RecyclerView) findViewById(R.id.portfolio_list);
+        mPortfolioListView = (RecyclerView)findViewById(R.id.portfolio_list);
+
+        mCursorAdapter = new CursorAdapter(this,null,0) {
```

Comment:
You can't use a CursorAdapter with a RecyclerView. Your adapter must extend RecyclerView.Adapter, which CursorAdapter does not.